appIcons.js

```diff
@@ -1856,27 +1856,27 @@ var MyShowAppsIconMenu = Utils.defineClass({
 
         if (this.sourceActor != Main.layoutManager.dummyCursor) {
             _appendItem(this, {
-                title: 'Power options',
+                title: _('Power options'),
                 cmd: ['gnome-control-center', 'power']
             });
 
             _appendItem(this, {
-                title: 'Event logs',
+                title: _('Event logs'),
                 cmd: ['gnome-logs']
             });
 
             _appendItem(this, {
-                title: 'System',
+                title: _('System'),
                 cmd: ['gnome-control-center', 'info-overview']
             });
 
             _appendItem(this, {
-                title: 'Device Management',
+                title: _('Device Management'),
                 cmd: ['gnome-control-center', 'display']
             });
 
             _appendItem(this, {
-                title: 'Disk Management',
+                title: ('Disk Management'),
```

Comment:
Did you forget the _ character?

Reply:
fixed by 7d26d0ae233ee54c3a8436c6e46974bbce465899